src/pyedb/configuration/cfg_boundaries.py

```diff
@@ -97,19 +97,25 @@ def apply(self):
         self._pedb.hfss.hfss_extent_info.use_open_region = self.open_region
         self._pedb.hfss.hfss_extent_info.open_region_type = self.open_region_type.name.lower()
         self._pedb.hfss.hfss_extent_info.is_pml_visible = self.pml_visible
-        self._pedb.hfss.hfss_extent_info.operating_freq = self.pml_operation_frequency
-        self._pedb.hfss.hfss_extent_info.radiation_level = self.pml_radiation_factor
+        if self.pml_operation_frequency:
+            self._pedb.hfss.hfss_extent_info.operating_freq = self.pml_operation_frequency
+        if self.pml_radiation_factor:
+            self._pedb.hfss.hfss_extent_info.radiation_level = self.pml_radiation_factor
         self._pedb.hfss.hfss_extent_info.extent_type = self.dielectric_extents_type.name.lower()
         if self.dielectric_base_polygon:
             self._pedb.hfss.hfss_extent_info.dielectric_base_polygon = self.dielectric_base_polygon
-        self._pedb.hfss.hfss_extent_info.dielectric_extent_size = float(self.horizontal_padding)
+        if self.horizontal_padding:
+            self._pedb.hfss.hfss_extent_info.dielectric_extent_size = float(self.horizontal_padding)
         self._pedb.hfss.hfss_extent_info.honor_user_dielectric = self.honor_primitives_on_dielectric_layers
         self._pedb.hfss.hfss_extent_info.extent_type = self.air_box_extents_type.name.lower()
         self._pedb.hfss.hfss_extent_info.truncate_air_box_at_ground = self.air_box_truncate_model_ground_layers
```

Comment:
@svandenb-dev You forget these ones?

Reply:
These are boolean you want also to don't touch ?

Reply:
@svandenb-dev  Boolean are also attributes. Do not touch.